testpilot/frontend/static-src/app/views/landing-page.js

```diff
@@ -37,7 +37,9 @@ export default PageView.extend({
     }
 
     app.sendToGA('pageview', {
-      'dimension1': this.loggedIn
+      'dimension1': this.loggedIn,
```

Comment:
Can we get this stuff documented in [README-METRICS.md](https://github.com/mozilla/testpilot/blob/master/docs/README-METRICS.md) - if not as part of this PR, then as part of a follow up in the very near future? Not sure what this dimension stuff is about, so I couldn't really speak to it in a review.